dspace-oai/src/main/java/org/dspace/xoai/app/XOAI.java

```diff
@@ -671,4 +682,15 @@ private static void usage() {
             System.out.println("     -h Shows this text");
         }
     }
+
+	public List<XOAIItemCompilePlugin> getxOAIItemCompilePlugins() {
```

Comment:
Ok, I will insert JavaDocs. No action on mark it as deprecated because we have to discuss first https://github.com/DSpace/DSpace/pull/2709#discussion_r393061152

Reply:
Same as other comments, either `@deprecated` or forward-ported